pkgs/applications/misc/3dslicer-binary/default.nix

```
@@ -0,0 +1,61 @@
+{stdenv, pkgs}:
+/*
+The source distribution uses cmake and seems to depend on some downloads and
+git repositories thus might be quite some work to package and update.
+Thus using binaries for now.
+*/
+
+stdenv.mkDerivation rec {
+
+  name = "slicer-binary-4.10.2";
+
+  src = pkgs.fetchurl {
+    name = "Slicer-4.10.2-linux-amd64.tar.gz";
+    url = http://slicer.kitware.com/midas3/download?bitstream=1023242;
+    sha256 = "1x92120v055qjs6yapjjpcp9wp4ajd9qpgdzgqbnk1j2c8nl5c31";
+  };
+
+  libPath = stdenv.lib.makeLibraryPath (
+    [
+      pkgs.stdenv.cc.cc
+      pkgs.pkgs.xorg.libX11
```

please remove pkgs usage
```suggestion
      stdenv.cc.cc
      xorg.libX11
```